pkg/remote/aws/s3_bucket_reader.go

```diff
@@ -0,0 +1,38 @@
+package aws
```

Comment:
I think you forgot to change filename after choosing the detailfetcher name ^^

Reply:
I was wondering, maybe can we put both enumerator and detail fetcher in a single `s3_bucket.go` file WDYT @wbeuil @moadibfr ?